components/dashboard/dashboard.html

```diff
@@ -1,5 +1,9 @@
 <div class="dashboard ng-cloak" ng-controller="DashboardController as model">
 
+  <div ng-if="model.state.redirectLink" class="row">
+    <div class="alert alert-info text-center col-md-offset-2 col-md-8">Go back to <a href="{{model.state.redirectLink}}">{{model.state.redirectLink}}</a> ?</div>
```

Comment:
Prepend text with: "It looks like you were referred here by another website. Go back to..."